qiskit_experiments/curve_analysis/standard_analysis/bloch_trajectory.py

```diff
@@ -0,0 +1,201 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Bloch trajectory analysis of single qubit."""
+
+from typing import List, Union, Optional
+
+import lmfit
+import numpy as np
+
+import qiskit_experiments.curve_analysis as curve
+import qiskit_experiments.data_processing as dp
+
+
+class BlochTrajectoryAnalysis(curve.CurveAnalysis):
+    r"""A class to analyze a trajectory of the Bloch vector of a single qubit.
+
+    # section: fit_model
+
+        The following equations are used to approximate the dynamics of the qubit Bloch vector.
+
+        .. math::
+
+            \begin{align}
+                F_x(t) &= \frac{1}{\Omega^2} \left(
+                    - p_z p_x + p_z p_x \cos(\Omega t') +
+                    \Omega p_y \sin(\Omega t') \right) + b \tag{1} \\
+                F_y(t) &= \frac{1}{\Omega^2} \left(
+                    p_z p_y - p_z p_y \cos(\Omega t') -
+                    \Omega p_x \sin(\Omega t') \right) + b \tag{2} \\
+                F_z(t) &= \frac{1}{\Omega^2} \left(
+                    p_z^2 + (p_x^2 + p_y^2) \cos(\Omega t') \right) + b \tag{3}
+            \end{align}
+
+        where :math:`t' = t + t_{\rm offset}` with :math:`t` is pulse duration to scan
+        and :math:`t_{\rm offset}` is an extra fit parameter that may represent the edge effect.
```

Comment:
```suggestion
        and :math:`t_{\rm offset}` is an extra fit parameter that may represent a edge effect.
```
Perhaps clarify what you mean by `edge effect`.

Reply:
Thanks, Done in https://github.com/Qiskit/qiskit-experiments/pull/824/commits/7ae1d01f51a226c6955382e7d0542b718bec7811